tests/testthat/test-rvar-print.R

```diff
@@ -1,75 +1,67 @@
 test_that("basic print.rvar works", {
-  skip_on_os("windows")
   x <- rvar(array(1:12, dim = c(2,2,3)))
   x_with_chains <- rvar(array(1:12, dim = c(2,2,3)), nchains = 2)
 
-  expect_output(print(rvar(), color = FALSE),
-"rvar<1>[0] mean ± sd:
-NULL",
-    fixed = TRUE
+  expect_output(
+    print(rvar(), color = FALSE),
+    "rvar<1>\\[0\\] mean . sd:\nNULL"
```

Comment:
this is also a bit difficult to read in tests, but should avoid the ± character matching issue.